src/main/java/org/testng/util/Strings.java

```diff
@@ -1,31 +1,35 @@
 package org.testng.util;
 
-import org.testng.collections.Lists;
 import org.testng.collections.Maps;
+import org.testng.internal.collections.Pair;
 
+import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 
-public class Strings {
+public final class Strings {
+  private Strings() {
+    //Utility class. Defeat instantiation.
+  }
   public static boolean isNullOrEmpty(String string) {
-    return string == null || string.trim().length() == 0; // string.isEmpty() in Java 6
+    return string == null || string.trim().isEmpty();
   }
 
   public static boolean isNotNullAndNotEmpty(String string) {
     return ! (isNullOrEmpty(string));
   }
 
-  private static List<String> ESCAPE_HTML_LIST = Lists.newArrayList(
-    "&", "&amp;",
-    "<", "&lt;",
-    ">", "&gt;"
+  private static List<Pair<String, String>> ESCAPE_HTML_LIST = Arrays.asList(
```

Comment:
Can we remove this list and only use the map?

Reply:
Fixed.